src/lib/parser-json-cov.cc

```diff
@@ -66,11 +66,11 @@ bool CovTreeDecoder::readNode(Defect *def)
         const pt::ptree &evtNode = it->second;
         DefEvent &evt = def->events[idx];
 
-        evt.fileName    = valueOf<std::string>(evtNode, "filePathname"    , "");
-        evt.line        = valueOf<int>        (evtNode, "lineNumber"      , 0 );
+        evt.fileName    = valueOf<std::string>(evtNode, "filePathname");
+        evt.line        = valueOf<int>        (evtNode, "lineNumber", 0);
```

Comment:
```suggestion
        evt.line        = valueOf<int>        (evtNode, "lineNumber");
```
Because `int() == 0`.